### DISCUSSION TITLE
New firmware not booting? (LCD screen blank)

### DISCUSSION THREAD

**Discussion by saxamaphoneman** [Jyers/Marlin#1756] [Q&A]
I just installed a BL touch for the first time. I have downloaded the jyers firmware that fits my mainboard (E3V2-BLTouch-3x3-v4.2.2-v1.3.4.bin). I've put the file on an empty SD card and put it in the printer, while it's off. Then turned the power switch on.

But nothing happens, I never get anything to appear on my LCD screen, it just looks like it's off. The printer turns on, as in the fans start up and the probe turns red and probes down twice, but the firmware doesn't appear to be loading. When I turn the printer off, take the SD card out and then turn it back on the screen still doesn't respond. 

Any suggestions on what I can do to fix the problem?

**Comment by MicroSur** [comment]
Try to flash LCD too.
https://github.com/Jyers/Marlin/tree/bugfix-JyersUI/Display%20firmware

**Reply by saxamaphoneman** [reply]
I followed the instructions and believe I got a successful flash (screen turned from blue to Orange). But after turning the machine off and back on, still, nothing displays on the LCD. Any further ideas?

**Reply by MicroSur** [reply]
Try to boot without SD card.
Try to use another SD card.

**Reply by saxamaphoneman** [reply]
Tried booting it without a card and it makes no difference. 

I won't be able to get my hands on a spare SD card until the weekend I don't think, but I'll try then (I vaguely recall reading somewhere not to use a card over 8gb? The only spares on hand are 32gb).

**Comment by Cal-LF** [comment]
You can revert to the original or any other firmware to get it back to running. It is important to have the SD card formatted to 4k fat32

**Reply by saxamaphoneman** [reply]
After reformatting the SD card to fat32 (which I think it was originally. Although I'm not sure what the 4K means and how to do this?) I just tried running the original firmware that I downloaded from creality and again, nothing happened. 

I might try open it up and check the ribbon cable didn't come disconnected on the main board, then I guess I'll follow MicroSur's succession and try flash the LCD

**Reply by saxamaphoneman** [reply]
Understood. Just did this making sure it was at 4K and still a blank LCD.

**Comment by Cal-LF** [comment]
See #1549 #1689

**Reply by saxamaphoneman** [reply]
I finally got my hands on a different SD card and it worked... Kind of. 

At first, I tried flashing the original Marlin from the Creality website and nothing happened. So I assumed a new SD card wasn't the fix. Then I tried flashing the jyers HS file instead of the one I was initially trying - and it worked! 

Would have been nice to know if the original jyers file worked, but I assumed it wouldn't since it didn't seem to be the SD card (or so I thought at that stage). Still a bit confused and slightly worried about why I can't flash the factory firmware?

One last question to confirm, does the HS just stand for high speed (of the probing I assume?) or is there anything else different about the HS file I need to be aware of?

Thanks to everyone who replied to me so far regarding this issue.

**Reply by tome9111991** [reply]
You can try to rename the firmware.bin and try to flash again.
Maybe it works with third try

**Reply by Cal-LF** [reply]
@saxamaphoneman HS = high speed. Happy to read you have some success.
Each time firmware is flashed it must have a different name. The old name is stored and will not accept it again.
Golden rule when changing firmware. Always reset eprom memory after flashing. Different firmware uses different eprom memory areas, even updates of the same firmware.

**Reply by MicroSur** [reply]
>(of the probing I assume?)

Yes.

**Reply by saxamaphoneman** [reply]
Thank you, needing to rename the firmware explains that behaviour then. Noted about resetting eprom settings, thanks once again
